integration_tests/run.sh

```diff
@@ -0,0 +1,10 @@
+export CGO_ENABLED=0
```

Comment:
@mjgarton got an opinion here?

Reply:
By the same token CGO isn't needed, therefore why enable it.

Reply:
I can't think of a reason why this would help in this instance, but in general if you don't need CGO, disabling it tends to avoid certain potential problems and rarely, if ever, causes problems.  Overall I have no strong view either way.